tally-power-normalization.ipynb

```diff
@@ -0,0 +1,366 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f6989d40",
+   "metadata": {},
+   "source": [
+    "Whenever we want our tallied quantities in units of \"per sec\" rather than \"per source neutron\", all we have to do is multiply by the normalization factor. For example, our fission rate would be:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "218ad621",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Fission rate = 9.715e+14 [reactions/sec]\n"
+     ]
+    }
+   ],
+   "source": [
+    "fission_rate_sec = fission_rate * source_per_sec\n",
+    "print(f'Fission rate = {fission_rate_sec:.3e} [reactions/sec]')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "19dd5d6c",
+   "metadata": {},
+   "source": [
+    "As a quick sanity check, we can multiply by 200 MeV/reaction to see how much heating we'd expect from this many fission reactions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "a6f5943a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Expected power = 31130.1 [W]\n"
+     ]
+    }
+   ],
+   "source": [
+    "expected_power = fission_rate_sec * 200e6 * joule_per_ev\n",
+    "print(f'Expected power = {expected_power:.1f} [W]')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f3364440",
+   "metadata": {},
+   "source": [
+    "The value is fairly close to the expected 30 kW but not quite exact since we used a crude estimate of 200 MeV/reaction. The actual amount of heat being deposited per reaction can be determined by dividing the heating rate by the fission rate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "19fbe062",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Heat per reaction = 192.7 [MeV]\n"
+     ]
+    }
+   ],
+   "source": [
+    "mev_per_reaction = heating_rate_ev / fission_rate * 1e-6\n",
+    "print(f'Heat per reaction = {mev_per_reaction:.1f} [MeV]')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d7b63bb5",
+   "metadata": {},
+   "source": [
+    "How much heat you observe will depend on which [score](https://docs.openmc.org/en/stable/usersguide/tallies.html#scores) you choose to estimate the heating. OpenMC has a variety of scores that allow you to tally heating that differ in subtle ways, so you should exercise some caution when picking a score. A few notes about the available scores:\n",
+    "\n",
+    "- `\"heating\"` — This score gives you neutron heating from *all* reactions, not just fission, so it gives you a way of accounting for heat from (n,$\\gamma$) reactions. Note that this score does not account for photon energy deposition and thus should generally be used only in a coupled neutron--photon calculation.\n",
```

Comment:
This wording is a bit confusing to me. Isn't the `"heating"` score giving you heating from all particles and all reactions?

> Note that this score does not account for photon energy deposition and thus should generally be used only in a coupled neutron--photon calculation.
>

reads in a confusing way to me, but I could be missing something.

Reply:
Ok, I just updated this. Hopefully this makes more sense now!